### ISSUE TITLE
False positive in nimble_operator rule

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#2489]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint --path file.swift --enable-all-rules --no-cache | grep nimble
Loading configuration from '.swiftlint.yml'
Linting Swift files at paths file.swift
Linting 'file.swift' (1/1)
Done linting! Found 2 violations, 0 serious in 1 file.
file.swift:2:2: warning: Nimble Operator Violation: Prefer Nimble operator overloads over free matcher functions. (nimble_operator)
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? 0.28.2
* Installation method used (Homebrew, CocoaPods, building from source, etc)? Homebrew
* Paste your configuration file:

```yml
# insert yaml contents here
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?
  If so, paste their relative paths and respective contents.

No

* Which Xcode version are you using (check `xcode-select -p`)?

10.1
* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
  to quickly test if your example is really demonstrating the issue. If your example is more
  complex, you can use `swiftlint lint --path [file here] --no-cache --enable-all-rules`.

```swift
// This triggers a violation:
foo.method {
    expect(value).to(equal(expectedValue), description: "Failed")
    return Bar(value: ())
}
```
